compiler-core/src/parse/error.rs

```diff
@@ -338,6 +338,16 @@ utf16_codepoint, utf32_codepoint, signed, unsigned, big, little, native, size, u
                 "This attribute cannot be used on a variant.",
                 vec!["Hint: Did you mean `@deprecated`?".into()],
             ),
+            ParseErrorType::PythonicImport { module, item } => (
+                "I was expecting either `/` or `.{` here.",
+                vec![
+                    "This syntax for an import is incorrect. Perhaps you meant:".into(),
+                    format!(" - `import {module}/{item}` to import the `{item}` \
+module from the `{module}` namespace"),
+                    format!(" - `import {module}.{{{item}}}` to import the \
+`{item}` module from the `{module}` module"),
```

Comment:
In `import {module}.{{{item}}}` the `{{{item}}}` is not a "module" but a value.
Example `import gleam.{io}` will not import the io module but give an error that "The module `gleam` does not have a `io` value".

```suggestion
`{item}` value from the `{module}` module"),
```

Reply:
Done :)